docs/index.rst

```diff
@@ -4119,17 +4122,21 @@ argument with code ``0`` to use ``last prediction`` and ``1`` for
     local_model.predict({"petal length": 3, "petal width": 1},
                         missing_strategy=PROPORTIONAL)
 
-If you would like a per-class prediction of the confidence or
-probability for each class in the model, you can use the
-``predict_probability`` and ``predict_confidence`` methods
-respectively.  Each of these methods takes the ``by_name`` and
-``missing_strategy`` arguments that function as in ``predict``, and
-one additional argument, ``compact``.
-
-If ``compact`` is ``False`` (the default), the output of these
-functions is a list of maps, each with the keys ``prediction`` and
-``probability`` mapped to the class name and its associated
-probability.
+For classification models, it is sometimes useful to obtain a
+probability or confidence prediction for each possible class of the
+objective field.  To do this, you can use the ``predict_probability``
+and ``predict_confidence`` methods respectively.  The former gives a
+prediction based a on the distribution of instances at the appropriate
+leaf node, with a Laplace correction based on the root node
+distribution.  The latter returns a lower confidence bound on the leaf
+node probability based on the Wilson score interval.
+
+Each of these methods takes the ``by_name`` and ``missing_strategy``
+arguments that function as in ``predict``, and one additional
```

Comment:
> arguments as in

I'm sorry, I'm not understanding the correction:  If I said:

...arguments that function as they do in ``predict``

would that be clear?